stackslib/src/chainstate/nakamoto/tests.rs

```diff
@@ -229,11 +387,11 @@ pub fn nakamoto_advance_tip_multiple() {
 
     let mut last_block: Option<NakamotoBlock> = None;
     let index_roots = [
-        "a8fa50e42ee48d393ac86036d8eec1bf6ada344de4100478fed37f1106281033",
-        "c431be733f37dee9dce70e6de51f616c2ed8b23583af8f7c4a8bce86c5f0a183",
-        "0c5e890e95e2f92ef36934bc0e5d71a6715974593f7d952b07ee6f959dae3f1c",
-        "0cd7696c4920ea5bc498ea46ee1df8566e06ea0bc8fd16a1e0ffd292d55f746e",
-        "84177188b1c02af772d2442b760fd9215b9dfadeed5723504acda2c94b068d15",
+        "c76d48e971b2ea3c78c476486455090da37df260a41eef355d4e9330faf166c0",
```

Comment:
I ended up changing these hashes when I was working on the coordinator (since I had to change some of the header structs, which changes the state root hash).  That's okay, right?

Reply:
Yeah, it's fine -- perhaps after factoring the mining routine in the mockamoto node, we should update this test anyways so that its able to figure out its own hashes.